apps/accounts/automations/gather_account/manager.py

```diff
@@ -289,6 +302,8 @@ def update_gather_accounts_status(self, asset):
                         "username": username,
                     }
                 )
+            risk_analyser = AnalyseAccountRisk(self.check_risk)
+            risk_analyser.lost_accounts(asset, lost_users)
 
         # 收集的账号 比 账号列表多的, 有可能是账号中删掉了, 但这时候状态已经是 confirm 了
         # 标识状态为 待处理, 让管理员去确认
```

Comment:
In your current code snippet, some of the changes and improvements can be suggested:

1. In the function `_create_risk` which has multiple parameters (`data`, `account`) instead of only one parameter.

   It could look like this:

```python
def _create_risk(self, asset, username, risk_choice=RiskChoice.unknown_account_value, details=[]):
```
   
2. The loss of accounts should also include detailed info (e.g. timestamp)

3. Update `lost_accounts` to follow naming conventions

4. Consider adding proper error handling for each part where possible to make program more robust
   
Remember to adjust according to existing context within your project.